packages/gatsby-plugin-image/src/components/hooks.ts

```diff
@@ -17,7 +17,7 @@ import { IGatsbyImageHelperArgs, generateImageData } from "../image-utils"
 const imageCache = new Set<string>()
 
 // Native lazy-loading support: https://addyosmani.com/blog/lazy-loading/
-export const hasNativeLazyLoadSupport =
+export const hasNativeLazyLoadSupport = (): boolean =>
```

Comment:
ℹ️ I've made this a function for lazy execution and not having code running on import but on demand